hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -1306,20 +1311,20 @@ public String getServerName() throws IOException {
 
   @Override
   public void startReconfigure() throws IOException {
```

Comment:
Maybe all OM/SCM/DN's `startReconfigure/getReconfigureStatus/listReconfigureProperties` can be implemented at `Class ReconfigurationHandler`

Reply:
Thanks @adoroszlai  for providing an elegant refactoring. I tested the PR in compose environment again and the result is as expected.
LGTM. +1